spec/coverage_helper.rb

```diff
@@ -0,0 +1,14 @@
+if ENV['CI']
+  require 'active_support/core_ext/kernel/reporting'
+
+  # Require all ruby files for accuracte test coverage reports
+  %w(app gems lib spa_ui).each do |path|
```

Comment:
Yeah. Though right now we're not tracking coverage for `gems/pending` at all, so maybe we don't need to care until those have been split/extracted?